src/ec/packages/ec-component-card/card.html.twig

```diff
@@ -78,7 +78,7 @@
 <article class="{{ _css_class }}"{{ _extra_attributes|raw }}>
   <header class="ecl-card__header">
     {% if _card.image.src %}
-      <div class="ecl-card__image" alt="{{ _card.image.alt }}" style="background-image:url('{{ _card.image.src }}')"></div>
+      <div class="ecl-card__image" alt-label="{{ _card.image.alt }}" role="img" style="background-image:url('{{ _card.image.src }}')"></div>
```

Comment:
wasn't this supposed to be an` aria-label`..? :D

Reply:
yes, I was too fast on this one :D Thx, done.